Xamarin.PropertyEditing.Tests/ObservableLookupTests.cs

```diff
@@ -56,7 +56,7 @@ public void RemoveLastItemInGroup (string key)
 			var grouping = lookup[key];
 			((INotifyCollectionChanged) grouping).CollectionChanged += (sender, args) => {
 				if (args.Action == NotifyCollectionChangedAction.Remove) {
-					if (args.OldItems[0] == value)
+					if ((string)args.OldItems[0] == value)
```

Comment:
Why?

Reply:
To remove compiler warnings like `Warning CS0252: Possible unintended reference comparison; to get a value comparison, cast the left hand side to type 'string' (CS0252) (Xamarin.PropertyEditing.Tests)`

Reply:
Btw, there are other warnings in the unit tests. If you'd rather we don't address them, I'll revert this change.